Momento/Incubating/Responses/CacheSetFetchResponse.cs

```diff
@@ -27,10 +29,15 @@ public CacheSetFetchResponse(_SetFetchResponse response)
             {
                 return null;
             }
-            return new HashSet<byte[]>(
+            if (_byteArraySet != null)
+            {
+                return _byteArraySet;
+            }
+            _byteArraySet = new HashSet<byte[]>(
                 elements.Select(element => element.ToByteArray()),
                 Utils.ByteArrayComparer
             );
+            return _byteArraySet;
```

Comment:
this works but are you aware of https://docs.microsoft.com/en-us/dotnet/framework/performance/lazy-initialization#implementing-a-lazy-initialized-property

also, if you do this as a guard it would be cleaner:

```suggestion
            if (_byteArraySet == null)
            {
                _byteArraySet = new HashSet<byte[]>(
                    elements.Select(element => element.ToByteArray()),
                    Utils.ByteArrayComparer
                );
            }
            return _byteArraySet;
```

Reply:
Awesome. Thanks for the lazy initialization pointer and reminder about using a guard. I committed lazy initializers to [59b036e](https://github.com/momentohq/client-sdk-csharp/pull/93/commits/59b036edf624045c1e3cf574556f29d33a3ab096)